solutions/gold/kattis-quantumsuperposition.mdx

```diff
@@ -28,66 +28,238 @@ Once we know all possible path lengths for each universe, we can find all
 possible sums of lengths. Just loop through both universe's route lengths and
 add them together.
 
+<LanguageSection>
+<CPPSection>
+
 ```cpp
 #include <bitset>
 #include <iostream>
 #include <queue>
 #include <vector>
 using namespace std;
 
+static const int MAX_N = 1001;
+static const int MAX_Q = 2001;
```

Comment:
why's the +1 necessary

Reply:
the graph is 1-indexed, and the dp needs states from [0, N] and [0, Q]

Reply:
can you make the graph 0-indexed
also use `MAX_N + 1` and `MAX_Q + 1` if you don't wanna

Reply:
alr i did the +1 thing